lib/tools/toolSystemMessage.tsx

```diff
@@ -42,7 +42,11 @@ const toolSystemMessage = (context: any, question: any, toolName: string) => {
 
     
     Ensure your answer is data-driven, insightful, and provides clear value for understanding and acting on the fan base's behavior.
-    Respond with a plain text string. Do not include any markdown formatting, JSON structure, or special characters. Avoid greetings, closings, or any meta-commentary about the response format.`;
+    Please provide a wide range of HTML formats with embedded HTML tags such as <div>, <p>, <ul>, <li>, and <span>, along with CSS styles including font size, margin, and padding. 
+    Please do not include any color styles. The font size for paragraphs should be 14px with left padding 8px & top padding 4px for indentation, while the font size for heading tags (h1 to h6) should be 16px and bold with top padding 4px.
+    All Numbers & Proper nouns should bold using <span>.
+    Make sure to present the HTML response as plain HTML without any enclosing code markers or delimiters.`;
```

Comment:
This same text is `duplicated` across multiple files.
- Could you please `DRY` this code into one, shared, const?
- ex name: `HTML_RESPONSE_FORMAT_INSTRUCTIONS`